src/dvr/dvr_config.c

```diff
@@ -810,7 +810,7 @@ const idclass_t dvr_config_class = {
     {
       .type     = PT_U32,
       .id       = "retention-days",
-      .name     = N_("DVR Log Retention Time (days)"),
+      .name     = N_("DVR Log retention period (days)"),
```

Comment:
I believe it means how long to keep the programme information in Tvheadend' database, @perexg can you confirm?